### DISCUSSION TITLE
Forge Server will not run

### DISCUSSION THREAD

**Discussion by benthyer** [itzg/docker-minecraft-server#1496] [Q&A]
Hi,

I have tried lots of different ways to get a forge server to run and I always fall at the same hurdle with the below error code.

```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 4 1000 1000 4096 Apr 30 21:27 /data'

2022-04-30T21:27:40.031Z	INFO	mc-server-runner	Done

2022-04-30T21:27:40.031Z	WARN	mc-server-runner	sub-process failed	{"exitCode": 1}

Error: An unexpected error occurred while trying to open file forge-1.12.2-14.23.5.2860.jar

mmap failed for CEN and END part of zip file
```

I am truly at a loss and the weird thing is that if the files are binded to a folder on my host and run in a docker volume instead, it works just fine?

Thanks,

Ben

**Comment by itzg** [comment]
My only guess is that the jar file download got interrupted. When you say

> if the files are binded to a folder on my host and run in a docker volume instead, it works just fine

That's two different volume modes. What's the third mode that isn't working for you?

**Reply by benthyer** [reply]
I have tried it multiple times with different versions and end up with the same error code.

I'm very new to servers, linux and dockers etc so I will use images to better explain.

The server that is working has this volume (I don't know how to access this file system to add mods)

![image](https://user-images.githubusercontent.com/61170294/166124567-3aaa22af-6ce9-4865-8a30-243b2ffbc4dc.png)

The server that isn't working has this setup so I can access the filesystems on the host pc

![image](https://user-images.githubusercontent.com/61170294/166124594-441a5302-9189-453d-866b-df28f1d6110f.png)

**Reply by itzg** [reply]
Thanks for the info. Are you running on Windows 11?

**Reply by benthyer** [reply]
No, it's a home server running on Ubuntu. Should've mentioned that earlier, apologies.

**Reply by itzg** [reply]
Thanks. I was only asking because Windows 11 specifically is having issues with corrupted files with host mounts.

Perhaps try deleting the downloaded jar file and starting the container again.

**Reply by benthyer** [reply]
Given that a go and still no luck, it's all very peculiar.

**Reply by benthyer** [reply]
I have replicated the setup for Paper Minecraft with the volume binded to an accessible folder structure on my host PC and that worked absolutely fine, just seems to be an issue with Forge at this stage.
